docs/linux/getting started/1. introduction.md

```diff
@@ -0,0 +1,47 @@
+### What is Linux Kernel?
+The Linux kernel is the main component of a Linux operating system (OS) and is the core interface between a computer’s hardware and its processes. It communicates between the two, managing resources as efficiently as possible.
+![Linux Kernel](/docs/linux/assets/img/linux_kernel.png "Linux Kernel")
+
+---
+### The Linux Foundation
+It's a non-profit organization that sponsors Linux and various other open source projects. 
+ -  Supported by individual members and large companies (Intel, TI, NXP, Oracle, etc.) 
+ -  Organize technical events and develop training programs 
+ -  Manages [`kernel.org`](https://www.kernel.org/) where the official versions of the Linux kernel are released 
+ -  Runs Linux web page [`linux.com`](https://www.linux.com/) 
+---
+### Linux Distributions & Families
+A Linux Distro is the combination of the kernel of Linux with other system components or programs to make a Linux based OS. The programs are used for:
+-   File-related operations
+-   User management
+-   Software package management.
+
+![Linux Distributions Tree](/docs/linux/assets/img/linux_distributions_tree.png "Linux Distributions Tree")
+
+ - FEDORA 
+    - Package manager → yum or/and dnf 
+    - Directly associated with Red Hat widely used by enterprises which host their own systems. 
+    - Examples → Fedora, RHEL (Red Hat Enterprise Linux), CentOS 
+ - DEBIAN 
+    - Widely used for servers and desktop 
+    - Focuses on stability 
+    - Package manager → DPKG-based apt-get 
+    - Ubuntu uses Unity, a GUI based on GNOME. It has a very small memory footprint and is widely used for cloud deployments. 
+    - Examples → Ubuntu 
+ - SUSE 
+    - Package manager → zypper 
+    - Widely used in the retail/sales sector (Ex → clothing stores, supermarkets). 
+    - Examples → SUSE, SUSE Linux Enterprise Server (SLES), openSUSE 
+---
+### Introduction
+-   Linux was written to be a free and open source version of UNIX.
+-   "All is a file" → Linux makes its components available via files or objects that look like files. So it can be used with the same utilities         used for regular files.
```

Comment:
```suggestion
-   "Everything is a file" → Linux makes its components available via files or objects that look like files. So it can be used with the same utilities used for regular files.
```